src/main/kotlin/no/nav/familie/ef/sak/integration/InfotrygdReplikaClient.kt

```diff
@@ -21,11 +23,22 @@ class InfotrygdReplikaClient(@Value("\${INFOTRYGD_REPLIKA_API_URL}")
     private val perioderOvergangsstønadUri: URI =
             UriComponentsBuilder.fromUri(infotrygdFeedUri).pathSegment("api/perioder/overgangsstonad").build().toUri()
 
-    fun hentPerioderOvergangsstønad(request: InfotrygdPerioderOvergangsstønadRequest)
-            : InfotrygdPerioderOvergangsstønadResponse {
+    private val eksistererUri: URI =
+            UriComponentsBuilder.fromUri(infotrygdFeedUri).pathSegment("api/stonad/eksisterer").build().toUri()
+
+    fun hentPerioderOvergangsstønad(request: InfotrygdPerioderOvergangsstønadRequest): InfotrygdPerioderOvergangsstønadResponse {
         return postForEntity(perioderOvergangsstønadUri, request)
     }
 
+    /**
+     * Infotrygd skal alltid returnere en stønadTreff for hver søknadType som er input
+     */
+    fun eksistererPerson(request: SøkFlereStønaderRequest): EksistererStønadResponse {
+        require(request.personIdenter.isNotEmpty()) { "Forventer att identer ikke er empty" }
+        require(request.stønader.isNotEmpty()) { "Forventer att søknadstyper ikke er empty" }
```

Comment:
svorsk-engelsk -> norsk for feilmeldingene? Forslag;
"Identer har ingen verdier"
"Søknadstyper har ingen verdier"